ingestion_service/producer/core/routers/api_endpoints_input.py

```diff
@@ -106,63 +110,70 @@ async def ingest_json(
         ], ):
     try:
         main_model_schema = jsoninput.json()
-
         serialized_message_json = json.dumps(main_model_schema)
         encoded_message_json = serialized_message_json.encode('utf-8')
         publish_message(encoded_message_json)
     except json.JSONDecodeError as e:
         raise HTTPException(status_code=400, detail="Invalid JSON" + str(e))
 
-    return JSONResponse(content={"message": "Data uploaded successfully"})
+    return JSONResponse(content={"message": "Data uploaded successfully to messaging server"})
 
 
 @router.post("/ingest/raw-jsonld",
-             # dependencies=[Depends(require_scopes(["write"]))]
+             dependencies=[Depends(require_scopes(["write"]))]
              )
 async def ingest_raw_jsonld(
```

Comment:
This function is listed as `async`, but I don't see any awaits, etc. called inside the function?